spec/controllers/taxons_controller_spec.rb

```diff
@@ -1,5 +1,3 @@
-require_relative "../../test/support/coronavirus_helper"
```

Comment:
just doing an initial glance at the code rather than a full, why is this being removed?

Reply:
because that helper's been removed, I'll show myself out

Reply:
Ah don't worry - I should have left a note in the commit. I think it was just an error that someone left this in the taxons_controller spec as that doesn't have anything to do with the coronavirus helpers.